src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -584,8 +590,6 @@ CHIP_ERROR OTARequestor::ExtractUpdateDescription(const QueryImageResponseDecoda
     update.softwareVersion = response.softwareVersion.Value();
 
     VerifyOrReturnError(response.updateToken.HasValue(), CHIP_ERROR_INVALID_ARGUMENT);
-    VerifyOrReturnError(response.updateToken.Value().size() >= 8 && response.updateToken.Value().size() <= 32,
```

Comment:
I think we should still check for the size, no? This guarantees that by the time we do the `CopySpanToMutableSpan` above, we are definitely not going to copy more than the buffer can handle.

Reply:
In CopySpanToMutableSpan we are checking size of buffer